components/style/gecko_bindings/structs_debug.rs

```diff
@@ -1176,7 +1175,10 @@ pub mod root {
         pub struct ThreadSafeAutoRefCnt {
             pub mValue: u64,
         }
-        pub const ThreadSafeAutoRefCnt_isThreadSafe: bool = true;
+        extern "C" {
+            #[link_name = "_ZN7mozilla20ThreadSafeAutoRefCnt12isThreadSafeE"]
+            pub static ThreadSafeAutoRefCnt_isThreadSafe: bool;
+        }
```

Comment:
Yeah, you're definitely using clang 3.8, but if you're not building with `--features llvm_stable`, so I expect stuff to break, like this, and potentially the `JSWhyMagic` thing.

I'm actually surprised it works so decently to be fair, but that's good :)